lib/id3c/cli/command/etl/presence_absence.py

```diff
@@ -175,6 +179,13 @@ def etl_presence_absence(*, db: DatabaseSession):
                     else:
                         identifier = f"NWGC/{received_sample_id}/{target.identifier}"
 
+                    # Look for clinical status -- historically, this detail was
+                    # stored at the top level, but with the new clinical
+                    # validation of OpenArray results, we may have a mix of
+                    # clinial and research results in one upload. Fall back to
+                    # the assayType if clinicalStatus is not available.
+                    clinical_status = test_result.get('clinicalStatus') or assay_type
```

Comment:
It might be easier for future debugging if the `assay_type` value gets determined in a single place. Could this line be moved to within `presence_absence_details()`?